pkg/generators/forwarding/fluentd/loki_conf.go

```diff
@@ -0,0 +1,30 @@
+package fluentd
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// LokiURL returns the endpoint url without any further processing
+func (conf *outputLabelConf) LokiURL() string {
+	url, err := url.Parse(conf.Target.Endpoint)
+	if err != nil {
+		log.Errorf("Failed parsing endpoint as a valid URL")
+		return ""
+	}
+
+	return fmt.Sprintf("%s://%s", url.Scheme, url.Host)
+}
+
+// LokiTenant returns the loki tenant ID
+func (conf *outputLabelConf) LokiTenant() string {
+	url, err := url.Parse(conf.Target.Endpoint)
+	if err != nil {
+		log.Errorf("Failed to extract Kafka topic from endpoint url %q: %s", conf.Target.Endpoint, err)
```

Comment:
I think something is wrong here.

Reply:
Or may be there is a relation with kafka :) no idea.

Reply:
No this really me switching between #425 and this PR :)